web_programming/instagram_crawler.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+import requests
+from bs4 import BeautifulSoup
+import json
+
+headers = \
+    {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36'}
+
+# Usage
+"""
+>>> user = Instagram("github")
+>>> user.is_verified()
+True
+>>> user.get_biography()
+Built for developers.
+
+"""
+
+
+class Instagram(object):
+    """
+    Class Instagram crawl instagram user information
+    """
+
+    def __init__(self, username):
+        self.username = username
+        self.url = 'https://www.instagram.com/{}/'.format(username)
+
+    def get_json(self):
+        """
+        return json of user information
+        """
+
+        html = requests.get(self.url, headers=headers)
+        soup = BeautifulSoup(html.text, 'html.parser')
+        try:
+            info = html_1(soup)
+            return info
+        except:
+            info = html_2(soup)
+            return info
```

Comment:
```suggestion
            return html_1(soup)
        except:  # <-- This repo does not accept bare excepts
            return html_2(soup)
```
Bare excepts are discussed in PEP8 and https://realpython.com/the-most-diabolical-python-antipattern/